src/news/release-1.3.0.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Release 1.3.0
+description: A lot of new things to check out!
+date: 2022-05-22
+tags:
+  - release
+---
+
+Hey!
+It's update-time again. This time we bring you some highlights, like Modrinth modpack support, a Windows installer, more paste services and improvements around our ATLauncher support.
+
+Initially this release should have included a workaround for download issues with CurseForge mods and modpacks, but we were contacted by CurseForge to remove our workaround.
+
+## Changelog
+
+### Added
+- Support ATLauncher share codes by @jamierocks in #408
+- Allow changing metadata-server settings by @dada513 in #425
+- Add PolyMC icon as instance icon by @ryanccn in #498
+- Add option to always hide Java setup wizard by @dada513 in #511
+- Add support for paste.gg, hastebin and mclo.gs paste services by @LennyMcLennington in #554
```

Comment:
Where's the CF key override here

Reply:
CF key override will most likely be reverted so we can't add it

Reply:
I assume it's milestone wasn't set, was wrong